app/models/ab_test_session.rb

```diff
@@ -0,0 +1,37 @@
+# reference document to map a user/metrics_uuid to a feature_flag A/B test group
+class AbTestSession
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  # use declarative labels so reports are clear what each group saw
+  GROUP_NAMES = %w[control experiment].freeze
```

Comment:
Non-blocking: "[treatment group](https://www.google.com/search?q=a%2Fb+testing+%22treatment+group%22)" looks like a 3.5x more popular way to refer to this thing than "[experiment group](https://www.google.com/search?q=a%2Fb+testing+%22experiment+group%22)".  The latter isn't uncommon, but it does seem drastically less common, so "treatment" strikes me as better.

```suggestion
  GROUP_NAMES = %w[control treatment].freeze
```

FWIW, my personal favorite here -- "[intervention group](https://www.google.com/search?q=a%2Fb+testing+%22intervention+group%22)" -- seems most common in statistics on public health and clinical trials.  It's slightly less prevalent than "treatment group" generally, though, and a fair amount less common in A/B testing material specifically.

Reply:
I think I like `intervention` as well - while `treatment` may be more prevalent it seems too clinical for this use case.